ftest.sh

```diff
@@ -304,6 +304,18 @@ if ${SETUP_ONLY:-false}; then
     exit 0
 fi
 
+# check if slurm needs to be configured for soak
+
+if [[ $TEST_TAG_ARG == *soak* ]]; then
+    IFS=, read -r -a CONTROL_NODE <<< $TEST_NODES
+    if ! ./slurm_setup.py -c ${CONTROL_NODE[0]} -n ${TEST_NODES} -s -i; then
```

Comment:
Does `slurrm_setup.py` need to be added in this PR?

Reply:
yes.   when any soak tag is used; I would like to make sure that slurm is configured/enabled

Reply:
Ahh.  It has already been committed.